.babelrc.js

```diff
@@ -0,0 +1,37 @@
+const env = process.env.BABEL_ENV || process.env.NODE_ENV || 'development';
+
+var pluginList = [
+  env === 'development' ? ['emotion', {sourceMap: true, autoLabel: true}] : 'emotion',
+  'transform-decorators-legacy',
+  'transform-class-properties',
+  'transform-object-rest-spread',
+  'transform-runtime',
+  'lodash',
+  'dynamic-import-node',
+  'syntax-dynamic-import',
+  'react-hot-loader/babel',
+  [
+    'babel-plugin-transform-builtin-extend',
+    {
+      globals: ['Array', 'Error'],
```

Comment:
i added Error here for no good reason other than I use it in Zeus and its nice to be able to extend it and maintain the type